package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.4",
   "versionCode": 4,
   "scripts": {
-    "start": "expo start --dev-client",
+    "start": "expo start",
```

Comment:
@confraria does this change make sense to you? @SirCQQ said without this the app wouldn't start locally

Reply:
The difference is that with `--dev-client` you need to build a native version of the app as it generates a url like this:
`com.nearform.optic://expo-development-client/?url=http%3A%2F%2F192.168.5.92%3A8081` so you have to run `expo run:android` first.

Without `--dev-client` you can just use expo go as the url it creates is `exp://192.168.5.92:19000`

As most things, it's a trade-off. Without `--dev-client` it's faster to start working as you don't have to install native tools but it hides some problems that only occur in the native versions like the google login problem and the qr code scan. 

We can remove it, add a `start:native` and update the README to reference these differences

Reply:
thanks for the explanation. let's use as default what's most convenient for local development please, and change the readme accordingly